app/Sources/TuistApp/TuistApp.swift

```diff
@@ -53,17 +55,24 @@ import TuistServer
                     CachedValueStore.$current.withValue(CachedValueStore(backend: .inSystemProcess)) {
                         Group {
                             if case .loggedIn = authenticationService.authenticationState {
-                                NavigationView {
-                                    PreviewsView()
-                                        .navigationBarItems(
-                                            trailing:
-                                            Button("Log Out") {
-                                                Task {
-                                                    await authenticationService.signOut()
-                                                }
-                                            }
-                                        )
+                                TabView {
+                                    NavigationView {
+                                        PreviewsView()
+                                    }
+                                    .tabItem {
+                                        NooraIcon(.deviceMobile)
+                                        Text("Previews")
```

Comment:
Should we make the content localizable? No need to do it as part of this PR though.

Reply:
Yeah, we should (could). But I don't think it's a priority right now.